### PR TITLE
Add script to grant writer permission to systems

### PR DESCRIPTION
**Pull request by credence0x**
PR for #113 
- Add script to grant writer permission to systems

### CODE REVIEW

File: contracts/scripts/system_components.json [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    "AttachCaravan": [
+        "Caravan",
+        "Movable",
+        "OrderId"
+    ],
+    "ClaimFungibleOrder": [
+        "Trade",
+        "Resource"
+    ],
+    "MakeFungibleOrder": [
```
Comment (aymericdelab) [inline contracts/scripts/system_components.json:37]: "Status" is missing here. Can you double check in all systems that no components are missing?

**Comment by aymericdelab** [comment]
Could you merge latest update from main in your code and then add the authorizations for "CancelFungibleOrder" as well ?

File: contracts/scripts/system_components.json (hunk 1 above)
Comment (credence0x) [reply contracts/scripts/system_components.json:37]: Updated and double checked

**Comment by credence0x** [comment]
> Great job on the code so far, could you also merge latest update from main in your code and then add the authorisations for "CancelFungibleOrder" as well ?

Thanks. Updated

**Review by aymericdelab — APPROVED** [review]

**Pull request merged**